protocol_tests/connection/test_automatic.py

```diff
@@ -0,0 +1,167 @@
+""" Manual Connection Protocol tests.
```

Comment:
With the pluggable backchannel PR merged, there should now be no need for separate manual and automatic tests.